quizgen/latex.py

```diff
@@ -37,10 +86,14 @@ def set_cli_args(parser):
                 + ' If not specified, $PATH will be searched.'
                 + ' Used to compile PDFs.'))
 
+    parser.add_argument('--use-docker', dest = 'use_docker',
+        action = 'store_true', default = False,
+        help = 'Use Docker to compile PDFs.')
     return parser
 
 def init_from_args(args):
-    if (args.pdflatex_bin_path is not None):
+    if args.use_docker:
+        set_pdflatex_use_docker(True)
+    if args.pdflatex_bin_path:
```

Comment:
NEVER (99.9% of the time) have two independent conditionals right next to eachother without whitespace between them.
It is very easy for a reader to accidentally read the second one as `elif` (since it is in the position an `elif` would be).